projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -237,5 +242,17 @@
     <igx-icon fontSet="material">drag_indicator</igx-icon>
 </ng-template>
 
+<ng-template #headSelectorBaseTemplate igxHeadSelector let-context>
+    <igx-checkbox 
+        [readonly]="true"
+        [checked]="context.totalCount === context.selectedCount"
```

Comment:
This may need a additional `context.selectedCount &&` check in case there's no data? I assume those would be 0-s then?

Reply:
![image](https://user-images.githubusercontent.com/3198469/64124614-21695480-cdb0-11e9-92b9-057bfabca0b5.png)